guides/common/modules/ref_storage-guidelines.adoc

```diff
@@ -31,9 +31,6 @@ endif::[]
 
 Log files are written to `/var/log/messages/,` `/var/log/httpd/`, and `/var/lib/foreman-proxy/openscap/content/`.
 You can manage the size of these files using *logrotate*.
-ifdef::satellite[]
-For more information, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/system_administrators_guide/ch-viewing_and_managing_log_files#s2-log_rotation[Log Rotation] in the _{RHEL} 7 System Administrator’s Guide_.
```

Comment:
Please replace this with the RHEL 8 equivalent.

Reply:
There doesn't seem to be an equivalent in RHEL8 Docs, but I was suggested this [Solution Article](https://access.redhat.com/solutions/1294) by people in the RHEL tech chat.

Reply:
Addressed :+1: 
Please review the wording.